### ISSUE TITLE
Add Watcher to Angular/Webpack Yarn Automation for Vendor and Custom SASS files

### ISSUE THREAD

**Issue by netblue** [jhipster/generator-jhipster#5779]
##### **Overview of the issue**

I'd like to be able to make more significant Bootstrap Style Mods incorporating modified Bootstrap and custom sass files.

##### **Motivation for or Use Case**

In my past Gulp based projects, I would incorporate a structure similar to below.  The vendor.scss would then reference my customBootstrap which would import from node or bower sources except for the specific components I wanted to override. 

e.g. src/main/webapp/content/scss
- bootstrap
-- _customBootstrap.scss
-- _navbar.scss
-- _variables.scss
-- _forms.scss
-- _modal.scss

##### **Reproduce the error**

It appears that vendor.scss and custom files are not specifically watched via the front end build automation.  (Unless I'm doing something wrong)  So I repeatedly stop yarn, manually run webpack:build:vendor and then restart yarn to make progress with iterative style changes.

##### **Suggest a Fix**
Unfortunately, I currently only know Gulp and not Webpack.  

##### **JHipster Version(s)**
4.4.1

##### **JHipster configuration**

{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.hof.app"
    },
    "jhipsterVersion": "4.4.1",
    "baseName": "hof",
    "packageName": "com.hof.app",
    "packageFolder": "com/hof/app",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "hibernateCache": "hazelcast",
    "clusteredHttpSession": "hazelcast",
    "websocket": "spring-websocket",
    "databaseType": "sql",
    "devDatabaseType": "mysql",
    "prodDatabaseType": "mysql",
    "searchEngine": "elasticsearch",
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "gradle",
    "enableSocialSignIn": true,
    "jwtSecretKey": "3bad1c6769840eacb42c1dcf17977409dad51988",
    "clientFramework": "angular2",
    "useSass": true,
    "clientPackageManager": "yarn",
    "applicationType": "monolith",
    "testFrameworks": [
      "gatling",
      "cucumber",
      "protractor"
    ],
    "jhiPrefix": "jhi",
    "otherModules": [
      {
        "name": "generator-jhipster-banner",
        "version": "0.0.2"
      }
    ],
    "enableTranslation": false
  }
}

##### **Browsers and Operating System**
Chrome on Mac OSX

- [ X] Checking this box is mandatory (this is just to show you read everything)

**Comment by xetys** [comment]
I hope I understood the issue correctly, feel free to correct me if I'm wrong.

At my work we use a private npm package, shared between a set of JHipster frontends, to make them look the same, by touching just one code base. This package is linked via. `yarn link` and included in several component styles and global.scss. Using this setup, any changes to the development version of our npm package triggers a hot reload in all running webpack processes.

I still assume you experience this issue because you imported that style in vendor.scss. I did run into this issue some time ago, too. We discussed that in #5039 and AFAIK it was closed like 'it is ok to be like this'.

I suggest you to try to move your vendor styles to global if you want them to hot reload.
